feature-ui-home/src/main/kotlin/team/duckie/app/android/feature/ui/home/screen/HomeFollowingScreen.kt

```diff
@@ -0,0 +1,252 @@
+package team.duckie.app.android.feature.ui.home.screen
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.DpSize
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import kotlinx.collections.immutable.PersistentList
+import team.duckie.app.android.feature.ui.home.R
+import team.duckie.app.android.feature.ui.home.common.RecommendUserProfile
+import team.duckie.quackquack.ui.color.QuackColor
+import team.duckie.quackquack.ui.component.QuackBody3
+import team.duckie.quackquack.ui.component.QuackDivider
+import team.duckie.quackquack.ui.component.QuackHeadLine2
+import team.duckie.quackquack.ui.component.QuackImage
+import team.duckie.quackquack.ui.component.QuackSubtitle2
+import team.duckie.quackquack.ui.component.QuackTitle2
+import team.duckie.quackquack.ui.modifier.quackClickable
+import team.duckie.quackquack.ui.util.DpSize
+
+internal data class Maker(
+    val cover: String,
+    val profile: String,
+    val title: String,
+    val name: String,
+    val takers: Int,
```

Comment:
```suggestion
    val takerNumber: Int,
```

takers 라 하면 시험에 응시한 유저 객체를 의미하는 것으로 보입니다.